pkg/controller/certificates/trigger/policies/gatherer_test.go

```diff
@@ -42,147 +41,100 @@ import (
+		wantCurCR  *cmapi.CertificateRequest
+		wantNextCR *cmapi.CertificateRequest
+		wantSecret *corev1.Secret
+		wantErr    string
 	}{
-		"the returned secret should stay nil when it is not found": {
+		"when no secret is found, the returned secret is nil": {
 			givenCert: gen.Certificate("cert-1", gen.SetCertificateNamespace("default-unit-test-ns"),
 				gen.SetCertificateSecretName("secret-1"),
 				gen.SetCertificateUID("uid-1"),
 			),
 			builder:    &testpkg.Builder{},
 			wantSecret: nil,
 		},
-		"the returned certificaterequest should stay nil when the list function returns nothing": {
+		"when no current or next CR exists, the returned cur and next CRs are nil": {
 			givenCert: gen.Certificate("cert-1", gen.SetCertificateNamespace("default-unit-test-ns"),
 				gen.SetCertificateSecretName("secret-1"),
 				gen.SetCertificateUID("uid-1"),
 			),
-			builder:     &testpkg.Builder{},
-			wantRequest: nil,
+			builder:   &testpkg.Builder{},
+			wantCurCR: nil,
 		},
-		"should find the certificaterequest that matches revision and owner": {
-			builder: &testpkg.Builder{
-				KubeObjects: []runtime.Object{},
-				CertManagerObjects: []runtime.Object{
-					gen.CertificateRequest("cr-4", gen.SetCertificateRequestNamespace("default-unit-test-ns"),
-						gen.AddCertificateRequestOwnerReferences(gen.CertificateRef("cert-1", "uid-4")),
-						gen.AddCertificateRequestAnnotations(map[string]string{
-							"cert-manager.io/certificate-revision": "4",
-						}),
-					),
-					gen.CertificateRequest("cr-7", gen.SetCertificateRequestNamespace("default-unit-test-ns"),
-						gen.AddCertificateRequestOwnerReferences(gen.CertificateRef("cert-1", "uid-7")),
-						gen.AddCertificateRequestAnnotations(map[string]string{
-							"cert-manager.io/certificate-revision": "7",
-						}),
-					),
-					gen.CertificateRequest("cr-9", gen.SetCertificateRequestNamespace("default-unit-test-ns"),
-						gen.AddCertificateRequestOwnerReferences(gen.CertificateRef("cert-1", "uid-9")),
-					),
-				},
-			},
-			givenCert: gen.Certificate("cert-1", gen.SetCertificateNamespace("default-unit-test-ns"), gen.SetCertificateNamespace("default-unit-test-ns"),
-				gen.SetCertificateUID("uid-7"),
-				gen.SetCertificateSecretName("secret-1"),
-				gen.SetCertificateRevision(7),
-			),
-			wantRequest: gen.CertificateRequest("cr-7", gen.SetCertificateRequestNamespace("default-unit-test-ns"),
-				gen.AddCertificateRequestOwnerReferences(gen.CertificateRef("cert-1", "uid-7")),
-				gen.AddCertificateRequestAnnotations(map[string]string{
-					"cert-manager.io/certificate-revision": "7",
-				}),
-			),
-		},
-		"should return a nil certificaterequest when no match of revision or owner": {
-			givenCert: gen.Certificate("cert-1", gen.SetCertificateNamespace("default-unit-test-ns"),
-				gen.SetCertificateUID("uid-1"),
-				gen.SetCertificateSecretName("secret-1"),
+		"when cert revision=1 and no owned CRs, the returned cur and next CRs should be nil": {
```

Comment:
Note you use `should be nil` here and `are nil` in the previous chunk

Reply:
I fixed "are nil" to "should be nil" in the above chunk. Thanks!